src/weapon/w_000.c

```diff
@@ -1,3 +1,8 @@
+// Weapon ID #0. Used by weapons:
+// Tyrfing, Namakura, Gladius, Scimitar, Cutlass, Saber, Falchion, Broadsword,
+// Bekatowa, Damascus sword, Hunter sword, Bastard sword, Talwar, Sword of
+// Hador, Luminus, Harper, Gram, Mormegil, Terminus Est, Dark Blade,
+// Mourneblade, Badelaire, Unknown
```

Comment:
Would it be useful to make the Unknowns `Unknown#XX` or something with the equipment ID?

Reply:
Good idea, done